doc/api/plotting/index.rst

```diff
@@ -27,6 +27,23 @@ all plotting functionality in PyVista.
 
    Plotter
    Renderer
+   Property
+
+
+Composite Plotting Reference
+----------------------------
+Composite dataset plotting reference. These classes are used when plotting
+composite datasets like :class:`pyvista.MultiBlock`.
```

Comment:
1. The first sentence seems redundant after the same heading.
2. What are "composite datasets _like_ [`MultiBlock`]" that aren't `MultiBlock`?

Reply:
We will eventually deprecate Multi-block datasets to composite datasets given the recent changes in VTK's API, but I think we're getting ahead of ourselves since it doesn't exist. Updated in aee6ffa58ee0f26d183b464f761ef056f110cae2